test/unit/download_test.rb

```diff
@@ -181,27 +181,32 @@ class DownloadTest < ActiveSupport::TestCase
   end
 
   should "find counts per day for versions in range across month boundary" do
-    Timecop.freeze(Time.zone.parse("2012-10-01")) do
-      @rubygem_1 = create(:rubygem)
-      @version_1 = create(:version, rubygem: @rubygem_1)
+    initial_time = Time.zone.parse("2012-10-01")
+    travel_to initial_time
+    @rubygem_1 = create(:rubygem)
+    @version_1 = create(:version, rubygem: @rubygem_1)
 
-      Timecop.freeze(1.day.ago) do
-        create :version_history, version: @version_1, count: 5
-      end
+    travel_to 1.day.ago do
+      create :version_history, version: @version_1, count: 5
+    end
 
-      Download.incr(@rubygem_1, @version_1.full_name)
+    travel_to initial_time
+    Download.incr(@rubygem_1, @version_1.full_name)
 
-      start = 2.days.ago.to_date
-      fin = Time.zone.today
+    start = 2.days.ago.to_date
+    fin = Time.zone.today
 
-      downloads = ActiveSupport::OrderedHash.new.tap do |d|
-        d[start.to_s] = 0
-        d["#{Time.zone.yesterday}"] = 5
-        d[fin.to_s] = 1
-      end
+    downloads = ActiveSupport::OrderedHash.new.tap do |d|
+      d[start.to_s] = 0
+      d["#{Time.zone.yesterday}"] = 5
+      d[fin.to_s] = 1
+    end
 
+    begin
       assert_equal downloads,
         Download.counts_by_day_for_version_in_date_range(@version_1, start, fin)
+    ensure
```

Comment:
This ensure should wrap all the way up, when we do the travel.. so if something else fail, we would make sure we travel back.
Maybe you can add a helper method here to wrap the entire thing with a block?